code/modules/surgery/organs/cell_holder.dm

```diff
@@ -0,0 +1,136 @@
+/obj/item/organ/internal/cell_mount
```

Comment:
Some proc to show additional information to the robot analyzer would be a good idea - that way, if a roboticist needs to find out what's gone wrong with a "Dead" IPC, they can see what's causing the IPC to not come back to life